src/Functional/Neq.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright (C) 2011-2016 by Lars Strojny <lstrojny@php.net>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+namespace Functional;
+
+use function Functional\not;
+
+/**
+ * verifies inequality.
+ *
+ * @param mixed $a
+ * @param mixed $b
+ * @return boolean
+ */
+function neq($a, $b)
+{
+    return $a != $b;
+
+    // OR
+    //return not('Functional\eq')($a, $b);
```

Comment:
ok with the best performing impl (`!=`).

Concerning strict equality (`!==`), I think we should provide both (which I did via the `is` method).

Concerning returning a function, I was wondering the same, but then when will we actually invoke the final function to get the result?